components/DataIntegrityProof.yml

```
@@ -12,13 +12,16 @@ info:
 paths:
 components:
   schemas:
-    LinkedDataProof:
+    DataIntegrityProof:
       type: object
-      description: A JSON-LD Linked Data proof.
+      description: A Data Integrity Proof as defined by the W3C VC Data Integrity specification.
       properties:
         type:
           type: string
-          description: Linked Data Signature Suite used to produce proof.
+          description: Data Integrity proof type.
```

"Proof" is capitalized in the object description so it might make sense to do the same here as well or to have it lower case for both
```suggestion
          description: Data Integrity Proof type.
```

---

They should be consistent, but I'm not sure whether titlecase `Proof` or lowercase `proof` is better.